mk/misc.mk

```diff
@@ -5,6 +5,7 @@ NVMEOF_CLI = $(DOCKER_COMPOSE_ENV) $(DOCKER_COMPOSE) run --rm nvmeof-cli --serve
 NVMEOF_CLI_IPV6 = $(DOCKER_COMPOSE_ENV) $(DOCKER_COMPOSE) run --rm nvmeof-cli --server-address $(NVMEOF_IPV6_ADDRESS) --server-port $(NVMEOF_GW_PORT)
 
 alias: ## Print bash alias command for the nvmeof-cli. Usage: "eval $(make alias)"
-	@echo alias cephnvmf=\"$(NVMEOF_CLI)\" \; alias cephnvmf-ipv6=\'$(NVMEOF_CLI_IPV6)\'
+	@echo alias cephnvmf=\"$(strip $(NVMEOF_CLI))\"
+	@echo alias cephnvmf-ipv6=\"$(strip $(NVMEOF_CLI_IPV6))\"
```

Comment:
`make` doesn't really care about leading whitespaces, and expressions like:
```make
NVMEOF_CLI := $(DOCKER_COMPOSE_ENV) ...
```
result in:
```sh
> make alias
alias cephnvmf=" docker-compose ..."
```
This whitespace before `docker-compose` matches a convention in most shell interpreters, where commands with a leading whitespace are explicitly excluded from the shell history (e.g.: running a command with a a secret/password).